src/App.js

```diff
@@ -21,7 +21,7 @@ class App extends Component {
         <NavBar />
         <Switch>
           <Route path="/" exact component={Home} />
-          <Route path="/game/:token" component={Game} />
+          <Route path="/game" component={Game} />
```

Comment:
I talked about this on slack, if you agree then we should change it back to using param. I can work on it if you don't have time